games-engines/gemrb/gemrb-0.8.5.recipe

```diff
@@ -0,0 +1,83 @@
+SUMMARY="An infinite game engine generator"
+DESCRIPTION="GemRB (Game Engine Made with preRendered Background) \
+is a port (actually a new implementation) of the original Infinity \
+Engine (the one of Baldur's Gate, Icewind Dale, Planescape: Torment, ...) \
+to Linux/Unix, MacOS X and Windows with some enhancements. Would \
+you like to create a game like Baldur's Gate?"
+HOMEPAGE="https://github.com/gemrb/gemrb"
+COPYRIGHT="2003-2017 The GemRB Team"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="https://github.com/gemrb/gemrb/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="d927e1eebd13a4643db363087fc61c5d2242e8a614da261f15bf4e2b1d94965c"
+SOURCE_DIR="gemrb-$portVersion"
+PATCHES="gemrb-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	gemrb = $portVersion
```

Comment:
```diff
 PROVIDES="
-	gemrb = $portVersion
+	gemrb = $portVersion$secondaryArchSuffix
```
Other changes are required to have a correct support for secondary architectures.
(You either need to add the `$secondaryArchSuffix` to the `cmd:` items, or define and use a `commandBinDir`.